### DISCUSSION TITLE
New wiki page 'Backup and restore'

### DISCUSSION THREAD

**Discussion by Sp1l** [dani-garcia/vaultwarden#1727] [Ideas]
Is there interest in a wiki page describing backup and restore of vaultwarden?

I'm not running Docker, but have vaultwarden running in a FreeBSD jail, storage connected to my MariaDB server (jailed).

I'm wary of failure modes with password managers... My (formerly Bruce Schneier's) [PasswordSafe](https://pwsafe.org) uses Yubikey protection and I've made sure I have 2 of them, one being the backup of the other. The vaults are also synced between various devices using Nextcloud etc...

I'll draft something below for review!

**Comment by Sp1l** [comment]
Arggghhhh... Hadn't looked, already exists! https://github.com/dani-garcia/vaultwarden/wiki/Backing-up-your-vault

**Reply by dumblob** [reply]
Improvements of all kind are always welcome! The restore part seems to need the most love as everybody is doing backup, but almost nobody is doing restoration. So any documented experience with restoration/recovery is a huge plus.
